corehq/apps/userreports/expressions/specs.py

```diff
@@ -314,6 +315,60 @@ def _get_forms(self, case_id, context):
         return xforms
 
 
+class FormsInDateExpressionSpec(JsonObject):
+    type = TypeProperty('get_case_forms_in_date')
```

Comment:
Should this be added to [UCR docs](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/userreports/README.md)?